presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPickTableLayout.java

```diff
@@ -47,6 +47,11 @@
     private TableHandle nationTableHandle;
     private TableLayoutHandle nationTableLayoutHandle;
 
+    public TestPickTableLayout()
+    {
+        super(true);
```

Comment:
Is this change intended to increase test coverage? Of what?

Also, it means we're not testing pick table layout with a connector that doesn't support push down?